Resources/Prototypes/_NF/Recipes/Lathes/prizecounter.yml

```diff
@@ -66,6 +66,31 @@
   result: PlushieLizard
   parent: BasePrize15Recipe
 
+- type: latheRecipe
+  id: Plushie_BotanistRecipe
+  result: Plushie_Botanist
+  parent: BasePrize15Recipe
+
+- type: latheRecipe
+  id: Plushie_CmoRecipe
+  result: Plushie_Cmo
+  parent: BasePrize15Recipe
+
+- type: latheRecipe
+  id: Plushie_ConstructionRecipe
+  result: Plushie_Construction
+  parent: BasePrize15Recipe
+
+- type: latheRecipe
+  id: Plushie_EngineerRecipe
+  result: Plushie_Engineer
+  parent: BasePrize15Recipe
+
+- type: latheRecipe
+  id: Plushie_SharkmMinnowRecipe
+  result: Plushie_SharkmMinnow
+  parent: BasePrize15Recipe
```

Comment:
Per the other emag comment, recommending to treat this in kind - move to the bottom with the other EMAG contents, change price to 50.